services/docker-compose.yml

```diff
@@ -124,6 +124,31 @@ services:
       - SUBDOMAIN_121_SERVICE=${SUBDOMAIN_121_SERVICE}
       - URL_PA_ACCOUNTS_SERVICE_API=${URL_PA_ACCOUNTS_SERVICE_API}
       - NODE_ENV=${NODE_ENV}
+      - ORMCONFIG_121_SERVICE_TYPE=${ORMCONFIG_121_SERVICE_TYPE}
+      - ORMCONFIG_121_SERVICE_HOST=${ORMCONFIG_121_SERVICE_HOST}
+      - ORMCONFIG_121_SERVICE_PORT=${ORMCONFIG_121_SERVICE_PORT}
+      - ORMCONFIG_121_SERVICE_USERNAME=${ORMCONFIG_121_SERVICE_USERNAME}
+      - ORMCONFIG_121_SERVICE_PASSWORD=${ORMCONFIG_121_SERVICE_PASSWORD}
+      - ORMCONFIG_121_SERVICE_DATABASE=${ORMCONFIG_121_SERVICE_DATABASE}
+      - ORMCONFIG_121_SERVICE_SCHEMA=${ORMCONFIG_121_SERVICE_SCHEMA}
+      - ORMCONFIG_121_SERVICE_ENTITIES=${ORMCONFIG_121_SERVICE_ENTITIES}
+      - ORMCONFIG_121_SERVICE_MIGRATIONSTABLE=${ORMCONFIG_121_SERVICE_MIGRATIONSTABLE}
+      - ORMCONFIG_121_SERVICE_MIGRATIONS=${ORMCONFIG_121_SERVICE_MIGRATIONS}
+      - ORMCONFIG_121_SERVICE_MIGRATIONSDIR=${ORMCONFIG_121_SERVICE_MIGRATIONSDIR}
+      - ORMCONFIG_121_SERVICE_DROPSCHEMA=${ORMCONFIG_121_SERVICE_DROPSCHEMA}
+      - ORMCONFIG_121_SERVICE_SYNCHRONIZE=${ORMCONFIG_121_SERVICE_SYNCHRONIZE}
+      - ORMCONFIG_121_SERVICE_LOGGING=${ORMCONFIG_121_SERVICE_LOGGING}
+      - ORMCONFIG_121_SERVICE_LOGGER=${ORMCONFIG_121_SERVICE_LOGGER}
+      - SECRETS_121_SERVICE_SECRET=${SECRETS_121_SERVICE_SECRET}
+      - USERCONFIG_121_SERVICE_PASSWORD_ADMIN=${USERCONFIG_121_SERVICE_PASSWORD_ADMIN}
+      - USERCONFIG_121_SERVICE_EMAIL_ADMIN=${USERCONFIG_121_SERVICE_EMAIL_ADMIN}
+      - USERCONFIG_121_SERVICE_PASSWORD_AID_WORKER=${USERCONFIG_121_SERVICE_PASSWORD_AID_WORKER}
+      - USERCONFIG_121_SERVICE_EMAIL_AID_WORKER=${USERCONFIG_121_SERVICE_EMAIL_AID_WORKER}
+      - USERCONFIG_121_SERVICE_PASSWORD_PROJECT_OFFICER=${USERCONFIG_121_SERVICE_PASSWORD_PROJECT_OFFICER}
+      - USERCONFIG_121_SERVICE_EMAIL_PROJECT_OFFICER=${USERCONFIG_121_SERVICE_EMAIL_PROJECT_OFFICER}
+      - USERCONFIG_121_SERVICE_PASSWORD_PROGRAM_MANAGER=${USERCONFIG_121_SERVICE_PASSWORD_PROGRAM_MANAGER}
+      - USERCONFIG_121_SERVICE_EMAIL_PROGRAM_MANAGER=${USERCONFIG_121_SERVICE_EMAIL_PROGRAM_MANAGER}
+      - USERCONFIG_121_SERVICE_COUNTRY_ID=${USERCONFIG_121_SERVICE_COUNTRY_ID}
```

Comment:
To simplify this whole list a bit, we should;
- only pass the variables (as there value is not changed in this `docker-compose`-file, right?  
See: https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers
- We can also pass the full `.env`-file by useing `env_file` instead of `environment`  
See: https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option